### PR TITLE
SW-1903 Show bagIds on View accession

### PR DESCRIPTION
**Pull request by constanzauanini**
Screen Shot 2022-10-04 at 14 42 56

Screen Shot 2022-10-04 at 13 00 29

### CODE REVIEW

File: src/components/accession2/view/Accession2View.tsx [hunk 1]
```diff
@@ -468,6 +468,12 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
             </Box>
           </Box>
         )}
+        {accession?.bagNumbers !== undefined && (
+          <Box display='flex' margin={isMobile ? themeObj.spacing(0, 0, 3, 6) : 0}>
+            <Typography fontWeight={600}> {strings.BAG_ID}: </Typography>
+            <Typography paddingLeft={1}> {accession.bagNumbers.join(', ')} </Typography>
```
Comment (charlie-tf) [inline src/components/accession2/view/Accession2View.tsx:474]: will this overflow to new lines if there are many bags with long names?

**Comment by karthikbtf** [comment]
The top row is starting to look so busy,, and uneven. Why this design? Why isn't this in the details section? We stick the plant ids in the details.

File: src/components/accession2/view/Accession2View.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/accession2/view/Accession2View.tsx:474]: Yes, it goes to new lines

**Comment by constanzauanini** [comment]
> The top row is starting to look so busy,, and uneven. Why this design? Why isn't this in the details section? We stick the plant ids in the details.

Is the design specified the bug https://terraformation.atlassian.net/browse/SW-1903

**Comment by karthikbtf** [comment]
> > The top row is starting to look so busy,, and uneven. Why this design? Why isn't this in the details section? We stick the plant ids in the details.
> 
> Is the design specified the bug https://terraformation.atlassian.net/browse/SW-1903

Yes, I understand that :) was just commenting on the decisions involved.
Looks like maybe we need less width for the edit icon on desktop (and spacefillers)? so the spacing across top row elements are even?

**Comment by constanzauanini** [comment]
> > > The top row is starting to look so busy,, and uneven. Why this design? Why isn't this in the details section? We stick the plant ids in the details.
> > 
> > 
> > Is the design specified the bug https://terraformation.atlassian.net/browse/SW-1903
> 
> Yes, I understand that :) was just commenting on the decisions involved. Looks like maybe we need less width for the edit icon on desktop (and spacefillers)? so the spacing across top row elements are even?

New spacings:
Screen Shot 2022-10-04 at 14 42 56

**Review by karthikbtf — APPROVED** [review]
Thanks 🎉 
Do you have a screenshot example for a long list of Bag IDs? just a sanity check...

**Comment by constanzauanini** [comment]
> Thanks 🎉 Do you have a screenshot example for a long list of Bag IDs? just a sanity check...

Screen Shot 2022-10-04 at 15 02 55

**Pull request merged**